resources/windows/wix/service.pxp-agent.wxs.erb

```diff
@@ -0,0 +1,45 @@
+<?xml version='1.0' encoding='windows-1252'?>
+<Wix xmlns='http://schemas.microsoft.com/wix/2006/wi' xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <!--
+  This is how to include wxi files.  We expect this file to define variables
+  like the version numbers and the upgrade UUID.
+  -->
+
+  <Fragment>
+    <DirectoryRef Id="PUPPETBINDIR">
+      <Component Id="service_nssm" Guid="52B1CD57-95A2-4CA4-AB8E-9DDD6DE8FC66">
+        <CreateFolder />
+        <File Id="NSSM" KeyPath="yes" Source="SourceDir\ProgramFiles64Folder\PuppetLabs\Puppet\bin\nssm.exe" />
```

Comment:
This should be `SourceDir\ProgramFiles64Folder\PuppetLabs\Puppet\puppet\bin\nssm.exe` as per https://github.com/puppetlabs/puppet-specifications/pull/63